src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator/help-checkoutCredentialsId.html

```diff
@@ -1,3 +1,15 @@
 <div>
-    <p>Credentials used to check out sources during a build.</p>
+    <p>
+        Credentials used to check out sources during a build.
+    </p>
+    <p>
+        The default choice of "Same as scan credentials" is sufficient so long as you don't
+        need Jenkins to push some changes back into these repositories.
+    </p>
+    <p>
+        Otherwise you should create an <strong>SSH username with private key</strong> credential and
```

Comment:
To quote @kohsuke "when in Rome, do as the Romans do".

GitHub recommends HTTPS (this recommendation seem to apply to both github.com and GitHub Enterprise), Atlassian BitBucket Server also reccomends HTTPS (for [other reasons](https://confluence.atlassian.com/bitbucketserver/enabling-ssh-access-to-git-repositories-in-bitbucket-server-776640358.html)).


I suspect that the usage of HTTPS will grow
This growth of HTTPS implies a better support of SSL certificate that are not trusted by JVM (and git executable) out-of-the-box: many JVMs do not support Let's Encrypt, it's common to use self-signed certificates (even though it has serious weaknesses, large organizations often have their own Root CA that is not declared in JVMs and in OS truststores by default...